b2b/api.py

```diff
@@ -793,3 +794,51 @@ def reconcile_user_orgs(user, organizations):
     caches["redis"].set(user_org_cache_key, sorted(orgs))
 
     return (len(contracts_to_add), len(contracts_to_remove))
+
+
+def reconcile_keycloak_orgs():
+    """
+    Reconcile Keycloak org records.
+
+    Retrieves the organizations for the configured realm out of Keycloak, and
+    create or update corresponding records in MITx Online. This does not manage
+    memberships, just base org info.
+
+    Returns
+    - tuple (created, updated): number of orgs created and updated
+    """
+
+    org_model = get_keycloak_model(*KCAM_ORGANIZATIONS)
+    orgs = org_model.list()
+    parent_org_page = OrganizationIndexPage.objects.first()
+    created = 0
+    updated = 0
+
+    for org in orgs:
+        page = OrganizationPage.objects.filter(sso_organization_id=org.id).first()
```

Comment:
Since this field isn't unique, should this just be a `get()` so we can catch if there's multiple pages w/ the same org id and log an error because that's an indication data needs to be fixed? Or should this field on `OrganizationPage` be unique?